src/places.js

```diff
@@ -0,0 +1,33 @@
+const getOr = require('lodash/fp/getOr')
+const googleMapsClient = require('@google/maps').createClient({
+  Promise,
+  key: process.env.GOOGLE_MAPS_API_KEY,
+})
+
+const getShortNameByType = (array, type) => {
+  const { short_name } = array.find(({ types }) => types.includes(type))
+  return short_name
+}
+
+exports.findPlaceInNantes = name =>( 
```

Comment:
La fonction est fail safe mais on veut pas plutôt qu'elle lance une erreur si rien n'est trouvé ?

Reply:
Elle n'est pas vraiment fail safe... La fonction getShortNamebyType peut tout à fait fail lors du destructuring. Je ne sais pas encore quoi faire si la recherche est infructueuse.

Dans une première itération on peut simplement tenter de récupérer les infos et afficher un message avertissement indiquant que nous n'avons pas réussit à récupérer les informations sur le lieu de l'évènement ? Qu'en penses-tu ?

Reply:
Oui très bien 😀